releases/v0.8.0.md

```diff
@@ -0,0 +1,38 @@
+# Release Notes for v0.8.0
+
+Release Date: Sep TBD, 2023
+
+Please see the [quickstart guide](../quickstart.md) for details on how to try out Confidential
+Containers.
+
+Please refer to our [Acronyms](https://github.com/confidential-containers/documentation/wiki/Acronyms)
+and [Glossary](https://github.com/confidential-containers/documentation/wiki/Glossary) pages for a
+definition of the acronyms used in this document.
+
+## What's new
+
+- TODO
```

Comment:
* Confidential Data Hub introduced on most platforms 
  * Supports sealed Kubernetes secrets that can be decrypted by KMS or stored in KBS
* Support added for multi-KBS model, where one KBS validates evidence using Attestation Service while another provides resource. 
* Something about policy @danmihai1
* Something host image pulling 
* Something about removing the containerd fork on most platforms

Reply:
> Something about policy @danmihai1

This is only `main` work.

Reply:
Oh yeah

Reply:
host image pulling with integrity(option) on most platforms(except s390x)
- support to download an image once on the host and sharing it as a block device with multiple pods or containers.
- not support downloading and sharing encrypted images on the host for the security of decryption keys.

Reply:
guest pulling image without forked containerd on most platforms:
-  support image pulling in VM without forked containerd, while utilizing a remote-snapshotter on the host without the connection to VM.
- support one remote snapshotter: nydus-snapshotter.